spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -267,31 +271,47 @@ private Dataset<Row> toDS(String schema, String jsonData) {
   }
 
   protected void validateDelete(
-      Snapshot snapshot, String changedPartitionCount, String deletedDataFiles) {
-    validateSnapshot(snapshot, DELETE, changedPartitionCount, deletedDataFiles, null, null);
+      int formatVersion, Snapshot snapshot, String changedPartitionCount, String deletedDataFiles) {
+    validateSnapshot(
+        formatVersion, snapshot, DELETE, changedPartitionCount, deletedDataFiles, null, null);
```

Comment:
rather than passing around the format version, would it make sense to parameterize `SparkRowLevelOperationsTestBase` and add `formatVersion` as another parameter?